src/docs/development/add-to-app/android/project-setup.md

```diff
@@ -4,4 +4,226 @@ short-title: Integrate Flutter
+            │   │   └── flutter_release-1.0.pom.sha1
+            │   ├── maven-metadata.xml
+            │   ├── maven-metadata.xml.md5
+            │   └── maven-metadata.xml.sha1
+            ├── flutter_profile
+            │   ├── ...
+            └── flutter_debug
+                └── ...
+```
+
+To depend on the AAR, the host app needs to be able to find these files.
+
+To do that, edit `app/build.gradle` in your host app such as it includes
+the local repository and the dependency:
+
+<?code-excerpt "MyApp/app/build.gradle" title?>
+```gradle
+android {
+  // ...
+}
+
+repositories {
+  maven {
+    url 'some/path/my_flutter/build/host/outputs/repo'
+    // This is relative to the location of the build.gradle file
+    // if using a relative path.
+  }
+  maven {
+    url 'http://download.flutter.io'
+  }
+}
+
+dependencies {
+  // ...
+  debugImplementation 'com.example.flutter_module:flutter_debug:1.0
+  profileImplementation 'com.example.flutter_module:flutter_profile:1.0
+  releaseImplementation 'com.example.flutter_module:flutter_release:1.0
+}
+```
+
+{{site.alert.tip}}
+You can also build an AAR for your Flutter module in Android Studio via
+the `Build > Flutter > Build AAR` menu.
+
+{% include app-figure.md image="development/add-to-app/android/project-setup/ide-build-aar.png" %}
+{{site.alert.end}}
+
+You can follow the next steps in the [API usage documentations](/docs/development/add-to-app/android/add-flutter-screen).
+
+#### 2. Depend on the module's source code
+
+This enables a one-step build for both your Android project and Flutter project.
+This is convenient when working on both parts simultaneously and rapidly
+iterate, but your team must install the Flutter SDK to build the host app.
+
+Include the Flutter module as a sub-project in the host app's `settings.gradle`:
+
+<?code-excerpt "MyApp/settings.gradle" title?>
+```groovy
+include ':app'                                     // assumed existing content
+setBinding(new Binding([gradle: this]))                                 // new
+evaluate(new File(                                                      // new
+  settingsDir.parentFile,                                               // new
+  'my_flutter/.android/include_flutter.groovy'                          // new
+))                                                                      // new
+```
+
+Assuming `my_flutter` is a sibling to `MyApp`.
+
+The binding and script evaluation allows the Flutter module to `include` itself
+(as `:flutter`) and any Flutter plugins used by the module (as `:package_info`,
+`:video_player`, etc) in the evaluation context of your `settings.gradle`.
+
+Introduce an `implementation` dependency on the Flutter module from your app:
+<?code-excerpt "MyApp/app/build.gradle" title?>
+```groovy
+dependencies {
+  implementation project(':flutter')
+}
+```
+
+You can follow the next steps in the [API usage documentations](/docs/development/add-to-app/android/add-flutter-screen).
```

Comment:
It might be helpful to point out to the reader that the existing Android app now includes the Flutter module and that the only thing left to do is launch a Flutter experience at the desired time and place. That would probably help frame for the reader what they've accomplished up to this point.

Reply:
Sounds good